src/main/java/org/jmhsrobotics/frc2023/subsystems/Drivetrain.java

```diff
@@ -248,6 +261,33 @@ public void resetOdometry(Pose2d pose) {
 		);
 		// spotless:on
 	}
+	public SequentialCommandGroup createRamseteCommand(Trajectory path) {
+		final DifferentialDriveKinematics driveKinematics = new DifferentialDriveKinematics(
+				AutoConstants.trackWidthMeters);
+
+		return new InstantCommand(() -> {
+			this.isInRamsete = true;
+		}).andThen(new RamseteCommand(path, this::getPose, new RamseteController(),
+				new SimpleMotorFeedforward(AutoConstants.trajectoryFeedForwardVolts,
+						AutoConstants.trajectoryFeedForwardVoltSecondsPerMeter,
+						AutoConstants.trajectoryFeedForwardVoltSecondsSquaredPerMeter),
+				driveKinematics, this::getWheelSpeeds,
+				new PIDController(AutoConstants.trajectorykP, AutoConstants.trajectorykI, AutoConstants.trajectorykD),
+				new PIDController(AutoConstants.trajectorykP, AutoConstants.trajectorykI, AutoConstants.trajectorykD),
+				this::tankDriveVolts, this)).andThen(new InstantCommand(() -> {
+					this.isInRamsete = false;
+				}));
+	}
+
+	public DifferentialDriveWheelSpeeds getWheelSpeeds() {
+		return new DifferentialDriveWheelSpeeds(leftFrontEncoder.getVelocity(), rightFrontEncoder.getVelocity());
+	}
+
+	public void tankDriveVolts(double leftVolts, double rightVolts) {
+		leftFrontMotor.setVoltage(leftVolts);
+		rightFrontMotor.setVoltage(-rightVolts);
```

Comment:
This code does not take into account motor inversions from other robots.
Something like this Might be more flexible. 
```java
rightFrontMotor.setVoltage((rightFrontMotor.getInverted() ? -1:1)*rightVolts);
```